proto/topology.proto

```diff
@@ -513,11 +589,17 @@ service TopologyService {
     rpc GetZone(ZoneRequest)                  returns (ZoneResponse);
     rpc ListPoolZone(ListPoolZoneRequest)     returns (ListPoolZoneResponse);
 
-
-    rpc CreatePhysicalPool(PhysicalPoolRequest)     returns (PhysicalPoolResponse);
-    rpc DeletePhysicalPool(PhysicalPoolRequest)     returns (PhysicalPoolResponse);
-    rpc GetPhysicalPool(PhysicalPoolRequest)        returns (PhysicalPoolResponse);
-    rpc ListPhysicalPool(ListPhysicalPoolRequest)   returns (ListPhysicalPoolResponse);
+    rpc CreatePhysicalPool(PhysicalPoolRequest)                        returns (PhysicalPoolResponse);
```

Comment:
Why are there so many spaces in front of return?

Reply:
> Why are there so many spaces in front of return?

redundant spaces are deleted